CHANGELOG.md

```diff
@@ -76,13 +76,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * `jj split` will now leave the description empty on the second part if the
   description was empty on the input commit.
 
-* Revsets gained a new function `mine()` that aliases `author([your_email])`.
-
 * `branches()`/`remote_branches()`/`author()`/`committer()`/`description()`
   revsets now support literal matching. For example, `branch(literal:main)`
   selects the branch named "main", but not "maint". `description(literal:"")`
   selects commits whose description is empty.
 
+* Revsets gained a new function `mine()` that aliases `author([literal:"your_email"])`.
```

Comment:
we'll see if we have to update this again if we rename the `literal:` prefix to `exact:` :)

Reply:
I was considering dropping the alias language, but I thought a simple example of the newly introduced literal syntax would be appreciated.